docs/src/inscriptions.md

```diff
@@ -106,3 +107,142 @@ A couple other endpoints that inscriptions may access are the following:
 - `/blockhash`: latest block hash.
 - `/blockhash/<HEIGHT>`: block hash at given block height.
 - `/blocktime`: UNIX time stamp of latest block.
+
+Provenance
+----------
+
+*NB: THIS SPEC IS NOT FINAL, NOT IMPLEMENTED, AND IS SUBJECT TO CHANGE AT ANY
+TIME. IT SHOULD NOT BE CONSIDERED FINAL UNTIL IT MAKE IT INTO A PUBLISHED,
+NUMBERED RELEASE OF `ord`. IF YOU CREATE INSCRIPTIONS BASED ON THIS IN-PROGRESS
+SPEC, IT WILL PROBABLY CHANGE AND YOU WILL PROBABLY GET REKT.*
+
+The holder of an inscription can create child inscriptions, trustlessly
+establishing the provenance of those children on-chain. This can be used for
+collections, with the children of a parent inscription being members of the
+same collection.
+
+Children can themselves have children, allowing for complex hierarchies. For
+example, an artist might create an inscription representing themselves, with
+sub inscriptions representing collections that they create, with the children
+of those sub inscriptions being items in those collections.
+
+### Specification
+
+To create a child inscription C with parent inscription P:
+
+- Create an inscribe transaction T as usual for C.
+- Spend the parent P in one of the inputs of T.
+- Include tag tag `3`, i.e. `OP_PUSH 3`, with the value of the serialized
+  inscription ID of P, `TXIDiINDEX`, serialized as the 32 byte binary `TXID`,
+  followed by the little-endian `INDEX`, with trailing zero bytes omitted.
```

Comment:
I think the savings in bytes by emitting the trailing zeros is not worth the extra complexities of handling those cases